internal/webserver/server.go

```diff
@@ -0,0 +1,123 @@
+//
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package webserver
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"runtime"
+
+	"github.com/edgexfoundry/app-functions-sdk-go/internal"
+	"github.com/edgexfoundry/app-functions-sdk-go/internal/common"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients"
+	logger "github.com/edgexfoundry/go-mod-core-contracts/clients/logging"
+
+	"github.com/gorilla/mux"
+)
+
+// WebServer handles the webserver configuration
+type WebServer struct {
+	Config        common.ConfigurationStruct
+	LoggingClient logger.LoggingClient
+	router        *mux.Router
+}
+
+// Test if the service is working
+func (webserver *WebServer) pingHandler(w http.ResponseWriter, _ *http.Request) {
+	w.Header().Set("Content-Type", "text/plain")
+	w.Write([]byte("pong"))
+}
+
+func (webserver *WebServer) configHandler(w http.ResponseWriter, _ *http.Request) {
+	webserver.encode(webserver.Config, w)
+}
+
+// Helper function for encoding things for returning from REST calls
+func (webserver *WebServer) encode(i interface{}, w http.ResponseWriter) {
+	w.Header().Add("Content-Type", "application/json")
+
+	enc := json.NewEncoder(w)
+	err := enc.Encode(i)
```

Comment:
Does this send the response? if so, add a comment stating that since it isn't ready apparent.